operations/companies.md

```diff
@@ -160,6 +173,13 @@ Adds a new company to the enterprise.
 | `Contact` | string | optional | Contact of the company. |
 | `Notes` | string | optional | Notes of the company. |
 | `Iata` | string | optional | Iata of the company. |
+| `Options` | [Company option parameters](#company-options-parameters) | optional | Options of the company. If not provided Invoiceable is set by default. |
```

Comment:
For adding it's required no?

Reply:
in code we have:
```
parameters.Options.MatchRef(o =>
            {
                company.Options = company.Options.WithFlag(CompanyOptions.Invoiceable, o.Invoiceable);
            });    
```

So you don't need to send the options, it would be a breaking change otherwise. Or am i missing something?

Reply:
One thing is implementation and one thing is documentation. In the documentation, you want to make it required.

Reply:
Okay then. Will update it.